ruby/vampires.rb

```diff
@@ -0,0 +1,71 @@
+
+#Employee Total
+puts "How many employees"
+employee_total= gets.chomp.to_i
+#Loop 
+employee=0
+until employee == employee_total
+
+#Name
+puts "what's your name"
+name=gets.chomp.capitalize!
+
+#Age
+puts "how old are you"
+age =gets.chomp.to_i
+
+#Year
+puts "What year were you born"
+year= gets.chomp.to_i
+if 2016 - year == age
+  year= true
+else 
+  year=false
+end
+
+#Garlicbread question
+puts "Do you want garlic bread Y or N"
+garlic_bread = gets.chomp
+if garlic_bread =="y" 
+   garlic_bread=true
+elsif garlic_bread =="n"
+   garlic_bread=false
+else puts "put y or n"
+end
+
+#Insurance Question
+puts "Do you need health insurance, put Y or N"
+insurance = gets.chomp.upcase!
+if insurance =="Y"
+  insurance= true 
+elsif insurance == "N"
+  insurance=false
+else puts "put Y or N"
+end
+
+#Allergies Loop
+puts "do you have any allergies, list them and enter done when done"
+allergies=gets.chomp
+until allergies == "done"|| allergies =="sunshine"
+   puts "list your alergies, enter done when done"
+   allergies =gets.chomp
+end
+  
+if name == "Drakula" || name == "TuFang"
```

Comment:
'Match the following conditions in the order they're listed.' We first read this when starting off the challenge and we may try to hack it out with a basic if/elsif structure but we quickly learn that the first condition to pass will be the result even if a 'more true' version is below.  We may be tempted to switch the order for a working solution, but if we are to follow the constraints we need to be more creative. What if we made each individual scenario it's own respective if block? What would go inside each if block? Surely we can't just puts out the result otherwise our screen would be full of each vampire check that passed. What we could do is keep setting a variable result within each if block and then at the end of all the blocks display the result to look something like this
```
if condition_1
	result = 'low'
end

if condition_2
	result = 'medium'
end

if condition_3
	result = 'high'
end

puts result
```